### DISCUSSION TITLE
Erro no build, Curso: Fundamentos do Azure, Git, GitHub e DevOps (49%)

### DISCUSSION THREAD

**Discussion by RafaelDifini** [balta-io/forum#94] [Tira Dúvidas] [answered]
Boa noite, estou tendo o seguinte problema ao tentar publicar o MySecondPrivatePackage com a referencia do MyCustomPrivatePackage, da erro no build do worflow

![image](https://user-images.githubusercontent.com/79666617/200957917-800cd935-1286-4480-9d6d-ede180afd2c3.png)

e aqui como esta o meu yml
![image](https://user-images.githubusercontent.com/79666617/200958072-54ebcf66-9ca5-4cb5-b6c6-30b78d18c210.png)

o erro parece dizer que nao encontrou o pacote na url, porem o pacote esta pubicado...
![image](https://user-images.githubusercontent.com/79666617/200958869-a0c44a84-3661-4193-bebd-e3559242dba6.png)

**Comment by andrebaltieri** [comment, marked as answer]
Duplicado
👉 https://github.com/balta-io/forum/discussions/84
